docs/source/models/causes/neonatal/preterm_birth.rst

```diff
@@ -154,17 +154,31 @@ Data Tables
 The Preterm Birth model requires only the probability of death (aka "mortality risk") for use
```

Comment:
As I mentioned in the meeting, we're not explicitly using the mortality risk in our modeling strategy, which could make the diagram a bit confusing. Perhaps we should change this wording. And as I suggested, maybe we should add a note below the tables in the graph section, explaining that the graph is more conceptual and is not meant to model the control flow of the program. And perhaps we could write down a formula for calculating the mortality risk if you want it.

Reply:
Also, I think maybe the entire description of the equations should go in a section called something other than "Data Tables", e.g., "Modeling strategy" or something, and the "Data Tables" section should just contain the table showing where to get the values. The order in which everything appears could remain the same, just add a section title and move the Data Tables section title.